src/app/components/publiclists/teamlist/teamcard/teamcard.component.ts

```diff
@@ -2,6 +2,7 @@ import { Component, OnInit, OnChanges, Input, Output, EventEmitter, SimpleChange
 import { GlobalService } from '../../../../services/global.service';
 import { ApiService } from '../../../../services/api.service';
 import { Router, ActivatedRoute } from '@angular/router';
+import { AuthService } from '../../../../services/auth.service';
```

Comment:
You imported this here, are you going to use this service? If yes then please add this service in the providers of spec file too else please remove this change @Kajol-Kumari

Reply:
> You imported this here, are you going to use this service? If yes then please add this service in the providers of spec file too else please remove this change @Kajol-Kumari

@Sanji515 Removed the import as it is not needed.